libevmasm/Instruction.h

```diff
@@ -188,6 +188,15 @@ enum class Instruction: uint8_t
 	SELFDESTRUCT = 0xff	///< halt execution and register account for later deletion
 };
 
+/// @returns true if the instruction is of the CALL opcode family
+inline bool isCallInstruction(Instruction _inst)
+{
+	return Instruction::CALL == _inst ||
+		Instruction::CALLCODE == _inst ||
+		Instruction::DELEGATECALL == _inst ||
+		Instruction::STATICCALL == _inst;
```

Comment:
Because like that not speaking me :D only Yoda.

Yah, I know the advantage, so we can't accidentally assign to var if we forget a `=`, but honestly, it reads so un-intuitive and shitty the const-first way in the majority of cases...